ubuntu_image/builder.py

```diff
@@ -208,7 +204,7 @@ def populate_bootfs_contents(self):
             target_dir = os.path.join(self.workdir, 'part{}'.format(partnum))
             # XXX: Use file system label for the moment, until we get a proper
             # way to identify the boot partition.
-            if part.filesystem_label == 'system-boot':
+            if part.role is StructureRole.system_boot:
```

Comment:
Please correct me if I'm wrong but we can get away with this because in the parser, you check to see if `role` is defined, and if not, you parse the file system label for fall backs, right?